lib/motor-controllers/spark-flex.ts

```diff
@@ -54,7 +59,7 @@ export const getMethods = () => ({
   getVelocityMethod: `return encoder.getVelocity() / gearRatio / 60.0; // Convert from RPM to RPS`,
 
   setPositionMethod: `double adjustedPosition = position * gearRatio;
-pidController.setReference(adjustedPosition, CANSparkBase.ControlType.kPosition, 0, 
+ProfiledPIDController.setReference(adjustedPosition, CANSparkBase.ControlType.kPosition, 0, 
```

Comment:
This doesn't work unless the  ProfiledPIDController is named ProfiledPIDController